tests/unit/amo/helpers.js

```diff
@@ -26,7 +26,10 @@ export const fakeAddon = Object.freeze({
     id: 123,
     license: { name: 'tofulicense', url: 'http://license.com/' },
     version: '2.0.0',
-    files: [],
+    files: [{
+      is_webextension: true,
+    }],
+    is_strict_compatibility_enabled: true,
```

Comment:
This should not default to to `true` in the shared `fakeAddon` object; it's an unlikely value